app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt

```diff
@@ -3,30 +3,22 @@ package fr.free.nrw.commons.utils
 import android.content.Context
 import android.content.pm.PackageManager
 import fr.free.nrw.commons.BuildConfig
-import java.util.*
 
 object ConfigUtils {
     @JvmStatic
-    val isBetaFlavour: Boolean
-        get() = BuildConfig.FLAVOR == "beta"
+    val isBetaFlavour: Boolean = BuildConfig.FLAVOR == "beta"
 
     @JvmStatic
-    private fun getVersionName(context: Context): String {
+    private fun Context.getVersionName(): String {
         return try {
-            context.packageManager
-                .getPackageInfo(context.packageName, 0).versionName
+            this.packageManager.getPackageInfo(this.packageName, 0).versionName
```

Comment:
`this` can be omitted in both extension functions

Reply:
Thanks, the extension functions are really cool.

Reply:
Dope as hell, you can also have extension properties which is identical to a function but you can access it like

```
context.versionName
```

from

```
val Context.versionName
get() = //code
```